gadgets/src/batched_is_zero.rs

```diff
@@ -89,6 +89,11 @@ impl<F: FieldExt, const N: usize> BatchedIsZeroChip<F, N> {
                 (F::one(), F::zero())
             }
         });
+
+        // annotate columns
+        region.name_column(|| "BATCHED_is_zero", config.is_zero);
+        region.name_column(|| "BATCHED_nonempty_witness", config.is_zero);
```

Comment:
Why `BATCHED`??

Reply:
Also, when annotating the same column twice, just the last annotation is preserved.

Reply:
my bad, the second one should be `config.nonempty_witness`. 

> Why BATCHED??
oh! should be `GADGETS_BATCHED`. There is another `is_zero` gadget so I only add prefix `BATCHED`. Do you  have any suggestions?